multigeiger/userdefines-example.h

```diff
@@ -21,6 +23,10 @@
 //#define   SERIAL_DEBUG Serial_One_Minute_Log  // One Minute logging
 //#define   SERIAL_DEBUG Serial_Statistics_Log  // Lists time [us] between two events
 
+// ** Special debug info: 
+//if set to 1, print debug info on serial (USB) interface while sending to servers (madavi or luftdaten)
```

Comment:
add a blank after the comment markup //.

Reply:
done